pkg/api/configsync/v1beta1/resource_override.go

```diff
@@ -66,6 +66,10 @@ type OverrideSpec struct {
 	// support pulling remote bases from public repositories.
 	// +optional
 	EnableShellInRendering *bool `json:"enableShellInRendering,omitempty"`
+
+	// logLevel allows one to override the log verbosity of a container
```

Comment:
Add more information to description here describing the expected format of the field

Reply:
added more description about the intended format of the field